docs/src/further_topics/netcdf_io.rst

```diff
@@ -235,3 +235,70 @@ Worked example:
     >>> my_coord.ignore_axis = True
     >>> print(guess_coord_axis(my_coord))
     None
+
+Multiple Coordinate Systems and Ordered Axes
+--------------------------------------------
+
+Traditionally, the coordinate system for coordinate of a data variable
+is specified by variable a single name in the ``grid_mapping`` attribute,
+e.g.:
+
+::
+
+    grid_mapping = 'latitude_longitude`
```

Comment:
Would be good to have a bit of surrounding context to make sure it's clear that were are talking about NetCDF variables, not Python variables (you will never see the `grid_mapping` attribute on an Iris variable).

Reply:
See 3e3a758ac